chain/network/src/store/schema/mod.rs

```diff
@@ -58,33 +57,6 @@ impl BorshRepr for ConnectionInfoRepr {
     }
 }
 
-#[derive(BorshSerialize, BorshDeserialize, ProtocolSchema)]
-pub(super) struct EdgeRepr {
```

Comment:
Curious, how/why did these and subsequent structs disappear?

Reply:
They're unused dead code.